rust_filter/src/graph_utils.rs

```diff
@@ -92,12 +96,36 @@ pub fn generate_trace_graph_from_headers(
             prev_node_handle = node_str_to_node_handle[node_str];
         }
     }
+
+    // now insert properties:
+    // do we have any?
+    let comma_index = properties_header.find(",");
+    if !comma_index.is_none() {
+        let properties_iterator = properties_header.split(",");
+        for statement in properties_iterator {
+            if !statement.find("==").is_none() {
+                let mut values_iterator = statement.split("==");
+                let property_with_node = values_iterator.next().unwrap().clone().to_string();
+                let mut period_iterator = property_with_node.split(".");
+                let node = period_iterator.next().unwrap();
+                let mut property = String::new();
+                for quality in period_iterator {
+                    property.push_str(&quality);
+                    property.push_str(".");
+                }
+                property.truncate(property_with_node.len()-node.len()-1); // we subtract an extra 2, one for the trailing period on the last iteration of the for loop
+                print!("node: {0}, property: {1}\n\n\n\n", node, property);
```

Comment:
Do we have logging facilities for rust?

Reply:
No, but per your comment on the tracing_sim repo, I think this makes more sense to maintain there.  So I'm going to put this as a TODO and then transfer it over instead of making all the Cargo.toml, etc, changes here.